app/src/main/java/app/gamenative/MainActivity.kt

```
@@ -501,6 +501,12 @@ class MainActivity : ComponentActivity() {
         if (level >= ComponentCallbacks2.TRIM_MEMORY_UI_HIDDEN) {
             releaseImageCaches()
         }
+
+        // Extremely aggressive for Legacy devices under critical pressure
+        if (level >= ComponentCallbacks2.TRIM_MEMORY_COMPLETE) {
+            appImageLoader?.diskCache?.clear()
+            System.gc()
```

detekt (1.23.8)

[warning] An explicit call to the Garbage Collector as in System should not be made.

(detekt.potential-bugs.ExplicitGarbageCollectionCall)